src/pages/registration/components/__tests__/RegistrationNotFound.test.tsx

```diff
@@ -0,0 +1,25 @@
+import { render, screen } from '@testing-library/react';
+
+import { MemoryRouter } from 'react-router';
+
+import { getCurrentTerm, getReadableTerm } from 'lib/utils/terms';
+import { renderWithRouter } from 'lib/utils/jest/index';
```

Comment:
if you run `npm run lint` it will tell you where your linting errors are,
`npm run lint:fix` will fix any that it is capable of, although it can't fix everything